### ISSUE TITLE
build RCall on aarch64 failed 

### ISSUE THREAD

**Issue by armgong** [JuliaInterop/RCall.jl#325]
on julia 1.3.0 master branch , build RCall failed on aarch64 linux
~~~julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.3.0-alpha.53 (2019-08-04)
 _/ |\__'_|_|_|\__'_|  |  Commit ca10039f80* (1 day old master)
|__/                   |

julia> using RCall
[ Info: Precompiling RCall [6f49c342-dc21-5d91-9882-a32aef131414]
ERROR: LoadError: LoadError: StackOverflowError:
in expression starting at /home/armgong/.julia/packages/RCall/iojZI/src/convert/missing.jl:2
in expression starting at /home/armgong/.julia/packages/RCall/iojZI/src/RCall.jl:42
ERROR: Failed to precompile RCall [6f49c342-dc21-5d91-9882-a32aef131414] to /home/armgong/.julia/compiled/v1.3/RCall/8GFyb.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1253
 [3] _require(::Base.PkgId) at ./loading.jl:1013
 [4] require(::Base.PkgId) at ./loading.jl:911
 [5] require(::Module, ::Symbol) at ./loading.jl:906

julia> versioninfo()
Julia Version 1.3.0-alpha.53
Commit ca10039f80* (2019-08-04 00:27 UTC)
Platform Info:
  OS: Linux (aarch64-unknown-linux-gnu)
  CPU: unknown
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, cortex-a53)


~~~

**Comment by terasakisatoshi** [comment]
- Similar problem occurs on my RaspberryPi4 (Ubuntu Server 20.04)
(R is installed via `apt install r-base`)

```
ubuntu@ubuntu:~$ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.4.1 (2020-04-14)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using RCall
ERROR: InitError: cfunction: closures are not supported on this platform
Stacktrace:
 [1] safe_parseVector(::Ptr{RCall.StrSxp}, ::Base.RefValue{Int32}, ::Ptr{RCall.NilSxp}) at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eval.jl:6
 [2] rparse_p(::Ptr{RCall.StrSxp}, ::Ptr{RCall.NilSxp}) at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eval.jl:49
 [3] rparse_p(::String, ::Ptr{RCall.NilSxp}) at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eval.jl:70 (repeats 2 times)
 [4] rparse(::String) at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eval.jl:74
 [5] set_hook(::String, ::RCall.RObject{RCall.LangSxp}) at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eventloop.jl:71
 [6] rgui_init() at /home/ubuntu/.julia/packages/RCall/Qzssx/src/eventloop.jl:79
 [7] __init__() at /home/ubuntu/.julia/packages/RCall/Qzssx/src/setup.jl:208
 [8] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [9] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:781
 [10] _require(::Base.PkgId) at ./loading.jl:1006
 [11] require(::Base.PkgId) at ./loading.jl:927
 [12] require(::Module, ::Symbol) at ./loading.jl:922
during initialization of module RCall

julia> versioninfo()
Julia Version 1.4.1
Commit 381693d3df* (2020-04-14 17:20 UTC)
Platform Info:
  OS: Linux (aarch64-unknown-linux-gnu)
  CPU: unknown
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, cortex-a72)

julia> Sys.ARCH
:aarch64
```

**Comment by randy3k** [comment]
> closures are not supported on this platform

It is the reason. We used a few closures in various places.

**Comment by simonbyrne** [comment]
Fixed by #441
